tests/cassettes/TestApiService.test_create_desk_is_successful.yaml

```diff
@@ -0,0 +1,27 @@
+interactions:
```

Comment:
What are these yaml files being used for?

Reply:
These are VCR cassettes for HTTP calls. How it works is that a real HTTP call is made only once and its request and response is recorded in these yaml files for subsequent test runs.

It basically allows us to test against the actual response that we'll receive from our API.

Reply:
Without having to actually interact with the API? Cool, didn't know this existed!

Reply:
Yup! It's heavily used when testing against 3rd party services in Ruby land as well!